src/test/crux-messenger/secure-cruxid-messenger-prod-test.ts

```diff
@@ -0,0 +1,62 @@
+import * as chai from "chai";
```

Comment:
any tests which interact with the server - put them in `src/test/integration-tests/crux-messenger/`

Reply:
done